### PR TITLE
fix XML comment formatting

### PR DESCRIPTION
**Pull request by J-N-K**
Signed-off-by: Jan N. Klug <jan.n.klug@rub.de>

### CODE REVIEW

**Comment by J-N-K** [comment]
Alphabetically?

**Comment by wborn** [comment]
Yes that's how they seem to be sorted now.

**Comment by J-N-K** [comment]
Done.

File: codestyle/src/main/resources/openhab_wst_xml_files.prefs [hunk 1]
```diff
@@ -1,6 +1,8 @@
 eclipse.preferences.version=1
+formatCommentJoinLines=false
 indentationChar=tab
 indentationSize=1
 lineWidth=120
 outputCodeset=UTF-8
 spaceBeforeEmptyCloseTag=false
+formatCommentJoinLines=false
```
Comment (wborn) [inline codestyle/src/main/resources/openhab_wst_xml_files.prefs:8]: One occurrence is usually enough.

**Pull request merged**